apps/design/frontend/src/image_input.tsx

```diff
@@ -158,14 +252,36 @@ export function ImageInput({
   disabled,
   className,
   required,
+  minWidthPx,
+  minHeightPx,
 }: {
   value?: string;
   onChange: (value: string) => void;
   buttonLabel: string;
   disabled?: boolean;
   className?: string;
   required?: boolean;
+  minWidthPx?: number;
+  minHeightPx?: number;
 }): JSX.Element {
+  const [error, setError] = useState<Error | undefined>(undefined);
+
+  // Clear error if the parent component has stopped interacting with this one
+  useEffect(() => {
+    if (disabled) {
+      setError(undefined);
+    }
+  }, [disabled]);
+
+  function onError(e: Error) {
+    setError(e);
+  }
```

Comment:
This doesn't actually prevent the form from being submitted. Is that intentional? If so, use a warning symbol and coloring?

Reply:
Whoops, left in `color: 'red'`. Meant to use a red consistent with the rest of the product.

I'll update this + form locking.

Reply:
Used native form validation and `Callout` instead of `P`: https://github.com/votingworks/vxsuite/pull/5936/commits/2a0354a0bbb26e2bc4bc891a086c18cd2e0bb3a3#diff-00afb2dfbef2411d546c95d8c995ca5938024b7b9ad8d483e59c43472b3ebd13R306